UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -398,12 +418,17 @@
         propertyDisplayValue = _.find(panels, function(panel){return panel.propertyValue == propertyValue.toString();}).propertyDisplayValue;
       }
 
-      var subTypesTrafficSigns = _.map(_.map(panels, function (group) {
+      var orderedPanels = sortByPropertyDisplay(panels);
+
+      var subTypesTrafficSigns = _.map(_.map(orderedPanels, function (group) {
+        var toHide = hideOldCodes(["147"], propertyValue, group.propertyValue);
```

Comment:
I think is better to move this value to the trafficSignColletion since it makes more sense there

Reply:
Done.

Reply:
Done